src/components/forecast-details.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import '../styles/forecast-details.css';
+
+const ForecastDetails = (props) => {
+return(
+  <div className="forcast-details">
+    <div>
+      <span>
+        <h5 className="date-more" data-testid="date-id">{moment(props.forecasts.date).format('ddd Do MMM')}</h5>
+      </span>
+    </div>
+
+    <div>
```

Comment:
when you have multiple div, span, h5 blocks think about how you could extract that into a reusable component and inject props to render your different properties